### ISSUE TITLE
Feature request: Run/Debug Commands

### ISSUE THREAD

**Issue by Richard87** [Haehnchen/idea-php-symfony2-plugin#1541]
Hi!

Thanks for an awesome plugin! It really saves me a lot of time every single day!

I would love a little green play button in the margin of commands, next to execute or the class...

Would that be easy to add / quick fix? maybe I could help?

**Comment by SergeC** [comment]
@Haehnchen Is any chance to have Run/Debug for Commands (same as for PHPUnit tests) in near future?

**Comment by Richard87** [comment]
I wonder if it will be way easier to make now if its based on the newish `#[AsCommand("app:command:name")` attribute!

**Comment by ChristianVermeulen** [comment]
@Haehnchen While it is awesome we can run commands straight from phpstorm, it's not ideal the way it is implemented right now. We work with docker containers and currently there is no way to configure the plugin to run the command inside a container, so for us the feature is not really useful at the moment.

However, it is in the way of easily clicking through to the configuration of the command. As you can see, the linemarker now replaces the cog for configuration.
![Screenshot 2022-05-12 at 10 28 04](https://user-images.githubusercontent.com/1062751/168027876-2ff2c1e3-c777-4a2a-82c5-b72dcfdefb1f.png)
![Screenshot 2022-05-12 at 10 28 15](https://user-images.githubusercontent.com/1062751/168027892-3a45fbef-8397-4e07-9d84-081e55fa454f.png)

Can we perhaps move it to to the `$defaulName`-line so we can support both?

**Comment by ChristianVermeulen** [comment]
My bad, turns out this specific command was not yet configured 🙈.

![Screenshot 2022-05-12 at 10 44 42](https://user-images.githubusercontent.com/1062751/168030300-a08bb2ab-7d98-4862-8481-9961df61c643.png)
![tenor-47887959](https://user-images.githubusercontent.com/1062751/168030560-f75b97a1-94b1-4b7b-a4fd-fecc1e670222.gif)

**Comment by Richard87** [comment]
Awesome! Thanks @Haehnchen !

Is it possible to also look for the Attribute `#[AsCommand('app:test')]` instead of `$defaultName`?

**Comment by Richard87** [comment]
![Uploading Skjermbilde 2022-05-12 kl. 12.04.37.png…]()
![Skjermbilde 2022-05-12 kl  12 05 06](https://user-images.githubusercontent.com/5749715/168046471-2a619f39-c47c-4575-a2e5-37b4942971d8.png)

I see you have added a codeblock for it (https://github.com/Haehnchen/idea-php-symfony2-plugin/pull/1920/files#diff-bd1d67aad9012c45bcb6b02f252d9f007cf79ae6b13cf0f5bbbb643df1b41569R73) but it doesnt seem to work...

I'm running Symfony plugin 2022.1.225, and PHPStorm 2022.1 :) 

Thanks for the amazing work, and is there anything I can do to help?

**Comment by Haehnchen** [comment]
thank you for this hint. i was thinking that the named arguments automatically were resolved by `PhpAttribute`, but its likely not the case.
`name: 'app:test:test'` should work.

i will fix this in the next release.

**Comment by Haehnchen** [comment]
@ChristianVermeulen yes, i was playing also with remote interpreter support like docker, but then stripped the first iteration, as this needed some improvements first, but will be added.

**Comment by Haehnchen** [comment]
Symfony command classes get a run gutter icon via `SymfonyCommandTestRunLineMarkerProvider`, and command run configurations are registered through `SymfonyCommandRunConfigurationType` and `SymfonyCommandRunConfigurationProducer`. Commands can
  be run through Symfony CLI or PHP/bin/console mode
